src/main/java/com/onevizion/scmdb/SqlScriptExecutor.java

```diff
@@ -158,4 +160,36 @@ private Connection getConnection(SchemaType schemaType, String schemaName) {
         }
     }
 
+    public void checkInvalidObjectAndThrow() {
```

Comment:
in https://github.com/Onevizion/scmdb/pull/88 was implemented common method to execute scripts, wait when it if will be merge to the master ans use it

Reply:
Updated this branch. Because of custom exception and exception message get from contex, i changed signature of methods  `execute()` - now they return context.